src/index.html

```diff
@@ -11,8 +11,56 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <!-- <h1>Product cards</h1> -->
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__product-picture"></div>
+
+      <div class="card__product-name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="card__product-code">Product code: 195434</div>
```

Comment:
use semantic tags for text content
**NOTE**: reset default `h2` and `p` margin similar you did with `body` tag
```suggestion
      <h2 class="card__product-name">
        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
      </h2>

      <p class="card__product-code">Product code: 195434</p>
```

Reply:
done